tcl.yaml

```diff
@@ -21,12 +21,11 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: 5880225babf7954c58d4fb0f5cf6279104ce1cd6aa9b71e9a6322540e1c4de66
+      expected-sha256: 861e159753f2e2fbd6ec1484103715b0be56be3357522b858d3cbb5f893ffef1
       uri: https://downloads.sourceforge.net/project/tcl/Tcl/${{package.version}}/tcl${{package.version}}-src.tar.gz
 
-  - uses: patch
-    with:
-      patches: include_stdint.patch
+  # Moved the include_stdint.patch here to avoid the need of update the hardcoded version in it on every update.
```

Comment:
This comment isn't very helpful after this PR merges. Not a huge deal just saying.

Reply:
removing, thanks!